metrodr-script.sh

```diff
@@ -12,43 +12,46 @@ scriptdir="$basedir/ramen-1"
 export PATH=${HOME}/.local/bin:${PATH}
 
 function check_for_command() {
-	if ! command -v $1 >/dev/null
+	if ! command -v "$1" >/dev/null
 	then
 		echo "command $1 not found"
 		exit 1
 	fi
 }
 
 function externalCluster_setFSID() {
-  local rook_tools_pod=$(kubectl --context "$1" -n $2 get pods -o name | grep tools)
-  ROOK_EXTERNAL_FSID=$(kubectl --context "$1" exec -n $2 ${rook_tools_pod} -- ceph fsid)
-  export ROOK_EXTERNAL_FSID
+        declare rook_tools_pod
+        rook_tools_pod=$(kubectl --context "$1" -n "$2" get pods -o name | grep tools)
+        ROOK_EXTERNAL_FSID=$(kubectl --context "$1" exec -n "$2" "${rook_tools_pod}" -- ceph fsid)
+        export ROOK_EXTERNAL_FSID
 }
 
 function externalCluster_setNamespace() {
-  if kubectl --context "$1" get ns -o name | grep -q "$2"
-  then
-	echo "namespace exists, continuing"
-    NAMESPACE="$2"
-    export NAMESPACE
-  else
-    kubectl --context "$1" create ns "$2"
-    NAMESPACE="$2"
-    export NAMESPACE
-  fi
+        if kubectl --context "$1" get ns -o name | grep -q "$2"
+        then
+                echo "namespace exists, continuing"
+        NAMESPACE="$2"
```

Comment:
The indentation here seems off. This and the next line should be at the same indentation level as the echo line above.

Reply:
corrected indentation level